packages/expo-eas-client/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### 🛠 Breaking changes
 
+- Removed deprecated Android `kotlin-android-extensions` plugin ([#19732](https://github.com/expo/expo/pull/19732) by [@josephyanks](https://github.com/josephyanks))
```

Comment:
I would put that in the `Others` category because it isn't a breaking change from the users' perspective.